web/src/narrow.js

```diff
@@ -782,6 +782,7 @@ export function update_selection(opts) {
     message_lists.current.select_id(msg_id, {
         then_scroll,
         use_closest: true,
+        mark_read: false,
```

Comment:
Generally, when we narrow, we do want to mark the selected message as read, so this change is not correct with the intended behavior.